driver.py

```diff
@@ -66,7 +66,7 @@ def translate_path(self, path):
     if len(sys.argv) > 1:
         site_mode = sys.argv[1]
 
-    addr = ("localhost", 8191)
```

Comment:
We intentionally switched to localhost after #1403

Reply:
Ha! I forgot about that. I'll switch it back and will just manually change it when I need to.